### ISSUE TITLE
About the loss

### ISSUE THREAD

**Issue by wangjiyuan9** [noahzn/Lite-Mono#103]
Hi, Author！Thank you for your amazing work!
I wonder the training loss would be at which scale when converged? If you have some training output log, it would be better!!

**Comment by wangjiyuan9** [comment]
I read the previous issue, and know that you didn't keep the loss log. So a range is ok. Be like [0.5(epoch 1) to 0.1 (epoch 20)] is ok!

**Comment by noahzn** [comment]
Hi, I can't remember the numbers. Please note that the depth losses are not accurate. See [here](https://github.com/noahzn/Lite-Mono/blob/main/trainer.py#L541).

**Comment by noahzn** [comment]
I am now closing this issue due to a lack of response.
